librz/core/cmd/cmd_shell.c

```diff
@@ -233,3 +233,33 @@ RZ_IPI RzCmdStatus rz_cmd_shell_fortune_handler(RzCore *core, int argc, const ch
 	rz_core_fortune_print_random(core);
 	return RZ_CMD_STATUS_OK;
 }
+
+static ut8 *getFileData(RzCore *core, const char *arg) {
+	if (*arg == '$') {
+		return (ut8 *)rz_cmd_alias_get(core->rcmd, arg, 1);
```

Comment:
shouldn't the alias be the argument of slurp ?

Reply:
I get that it compares the contents of two aliases, not sure why.

Reply:
if you read what that command used to do, it says that it use the value stored in the alias as the filename of the bin to diff

Reply:
`"dif", " [$a] [$b]", "Compare two files (or $alias files)",`

Reply:
i guess this was never tested. i'm ok to remove this feature and just diff 2 files.

Reply:
Removed alias diffing then.